File: angular6-client/src/app/_model/Stock.ts
export class Stock {
	public stockId: string;
	public symbol: string;
	public price: string;
	public trend: string;

	constructor(stockId: string, symbol: string, price: string, trend: string) {
		this.stockId = stockId;
		this.symbol = symbol;
		this.price = price;
		this.trend = trend;
	}

	public static default(): Stock {
		return new Stock("", "", "", "");
	}
}

File: src/main/java/com/colin/app/entity/model/Stock.java
package com.colin.app.entity.model;

/**
 * This model to map stock client data
 * 
 * @author colinle
 *
 */
public class Stock {
	private String symbol;

	private String price;

	private String trend;

	public Stock(String symbol, String price, String trend) {
		this.symbol = symbol;
		this.price = price;
		this.trend = trend;
	}
}
